### DISCUSSION TITLE
Default parameters in MS2 Rescore

### DISCUSSION THREAD

**Discussion by TheB1307** [CompOmics/ms2rescore#164] [Q&A] [answered]
Dear community

I am using MS2 Rescore for processing my data which were identified with Sage. There I used 100 % FDR. Now, I run MS2 Rescore and I am wondering how high the FDR is. I tried to find a solution in the output files and in the paper which were published. But I can't find an answer. Also if I am able to change the number or it is by default always 1 %?  Can somebody help me with that? 

Have a great day!

**Comment by RalfG** [comment]
Hi @TheB1307,

Thanks for the question! MS²Rescore always returns all PSMs without FDR filtering (so 100%). The `ms2rescore.psms.tsv` file has a q-value column that can be used to filter the PSMs. For instance, only keeping rows with q-value < 0.01 will result in a file that is filtered on 1% FDR. Note that decoy PSMs are also included.

The same is true for the `mokapot.psms.txt`, `mokapot.peptides.txt` and `mokapot.proteins.txt` files, although these only contain target PSMs with decoy PSMs having their own respective text files.

We can easily add a configuration parameter to write only target PSMs filtered at an FDR threshold. I will open an issue for this.

Best,
Ralf

**Reply by TheB1307** [reply]
Hi @RalfG 

Thank you so much for your answer! That's really helpful. And thank you for adding the additional parameter :) I am using the function to write a FlashLQF-file so I directly can use it for the next step. Would be great to directly use the file instead of an additional filtering step.

Have a great day!
Best wishes!

**Reply by RalfG** [reply, marked as answer]
Hi @TheB1307,

Do you use MS²Rescore's built-in output for FlashLFQ? This enables Mokapot's [`to_flashlfq`](https://mokapot.readthedocs.io/en/latest/api/functions.html#mokapot.to_flashlfq) function, which by default only contains confident target peptides. So no further filtering is required on this output file.

Best,
Ralf

**Reply by TheB1307** [reply]
Hi @RalfG 
Yes, I use this! That sounds great! thank you so much for the answer.
Best!
